server/controllers/routes/user/cohort/getSpecificCohort.js

```diff
@@ -0,0 +1,13 @@
+const {
+  cohort: { getSpecificCohort },
+} = require('../../../../database/queries');
+
+exports.getSpecificCohort = async (req, res, next) => {
+  try {
+    const { rows } = await getSpecificCohort(req.params.cohortid);
+    const data = { ...rows[0] };
+    res.json({ statusCode: 200, data });
```

Comment:
I  sent a request with cohortId 10  and response data was null, so here you need to Handle error if the cohort does not exist !!